setup.py

```diff
@@ -12,39 +12,46 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from __future__ import print_function
 import os
 
-readme_filename = os.path.join(os.path.dirname(__file__), 'README.md')
-with open(readme_filename, 'r') as f:
-  readme = f.read()
+from setuptools import setup
+
+readme_filename = "README.md"
+current_directory = os.path.dirname(__file__)
+readme_path = os.path.join(current_directory, readme_filename)
 
 try:
-  import pypandoc
-  readme = pypandoc.convert(readme, to='rst', format='md')
+    with open(readme_path, 'r') as f:
+        readme = f.read()
 except:
-  print("Conversion of long_description from MD to reStructuredText failed...")
-  pass
+    print("Failed to open %s" % readme_path)
 
+try:
+    import pypandoc
+    readme = pypandoc.convert(readme, to='rst', format='md')
+except:
+    print("Conversion of long_description from MD to reStructuredText failed...")
```

Comment:
minor suggestion: when catching all exceptions (i.e. a bare "except" clause), it's a future debugging time saver to include the exception string repr in the error message.